### PR TITLE
Side bar added and simple functionality

### PR DESCRIPTION
**Pull request by Abdul0046**

### CODE REVIEW

**Comment by lucas-homer** [comment]
For frontend PRs, we always include some sort of visual aid that shows the changes to the UI. Most of the time this takes the form of a Loom video, and in simpler cases a screenshot is fine. I just sent you an invite for Loom to your selectquote email so you can get set up with that. The video doesn't need to be anything too complicated. Simply demonstrate what the PR's code adds. Also -- we have a doc for "what makes a good PR" https://selectquote.atlassian.net/wiki/spaces/SE/pages/839745693/What+makes+a+Great+PR

**Comment by Abdul0046** [comment]
> For frontend PRs, we always include some sort of visual aid that shows the changes to the UI. Most of the time this takes the form of a Loom video, and in simpler cases a screenshot is fine. I just sent you an invite for Loom to your selectquote email so you can get set up with that. The video doesn't need to be anything too complicated. Simply demonstrate what the PR's code adds. Also -- we have a doc for "what makes a good PR" https://selectquote.atlassian.net/wiki/spaces/SE/pages/839745693/What+makes+a+Great+PR

Sure I'll make one

File: src/components/SideNavBar.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import PeopleIcon from '@material-ui/icons/People';
+import SideNavIcons from '@/components/SideNavIcons';
+import { SideNav, SideNavMiddle, SideNavTop } from 'scplus-shared-components';
+import AssignmentIndIcon from '@material-ui/icons/AssignmentInd';
+
+const SideNavBar = () => {
+  return (
+    <SideNav>
+      <SideNavTop>
+        <SideNavIcons>
```
Comment (lucas-homer) [inline src/components/SideNavBar.js:12]: [blocker] I would've expected an IconButton component here. We should review the multi-instance-admin-view repo for guidance

File: src/components/SideNavBar.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            onClick={() => {
+              alert('I am clicked');
+            }}
+          >
+            A
+          </span>
+        </SideNavIcons>
+      </SideNavTop>
+      <SideNavMiddle>
+        <SideNavIcons>
```
Comment (lucas-homer) [inline src/components/SideNavBar.js:23]: similar comment here. We should review our example repo (multi-instance-admin-view) for guidance on how to build a sidenav and buttons in the sidenav

**Review by lucas-homer — CHANGES_REQUESTED** [review]
We should abandon SideNavIcons and review the MIAV repo for reference/guidance. Feel free to respond with any questions about that you may have

File: src/components/SideNavBar.js (hunk 1 above)
Comment (Abdul0046) [reply src/components/SideNavBar.js:12]: Got it. I will use IconButtons for this.

File: src/components/SideNavBar.js (hunk 2 above)
Comment (Abdul0046) [reply src/components/SideNavBar.js:23]: I will surely look into the code. Actually, I tried to build it by myself so I didn't refer to multi-instance admin view

Comment (Abdul0046) [reply src/components/SideNavBar.js:23]: If you are referring to that SideNavIcons that's a component I made

**Pull request merged**